ceph/rados/core_workflows.py

```diff
@@ -3849,3 +3849,16 @@ def get_osd_details(self, osd_id: int):
         details = out["nodes"][0]
         log.debug(f" OSD : {osd_id} details are : {details}")
         return details
+
+    def get_osd_size(self, osd_id):
+        """
+        Method returns the size of the OSD
```

Comment:
maybe just elaborate in the docstring that the size is returned in KB

Reply:
Added